recipes/zbar/all/conanfile.py

```diff
@@ -0,0 +1,100 @@
+                       'with_xv': False,
+                       'with_jpeg': False,
+                       'enable_pthread': True,
+                       }
+
+    _env_build = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def _configure_autotools(self):
+        if not self._env_build:
+            self._env_build = AutoToolsBuildEnvironment(self)
+            env_args = []
+            env_args.extend([
+                "--enable-video" if self.options.with_video else "--disable-video",
+                "--with-imagemagick" if self.options.with_imagemagick else "--without-imagemagick",
+                "--with-gtk" if self.options.with_gtk else "--without-gtk",
+                "--with-qt" if self.options.with_qt else "--without-qt",
+                "--with-python" if self.options.with_python_bindings else "--without-python",
+                "--with-x" if self.options.with_x else "--without-x",
+                "--with-xshm" if self.options.with_xshm else "--without-xshm",
+                "--with-xv" if self.options.with_xv else "--without-xv",
+                "--with-jpeg" if self.options.with_jpeg else "--without-jpeg",
+                "--enable-pthread" if self.options.enable_pthread else "--disable-pthread",
+                "--enable-shared" if self.options.shared else "--disable-shared",
+                "--enable-static" if not self.options.shared else "--disable-static",
+            ])
+            self._env_build.configure(args=env_args, configure_dir=self._source_subfolder)
+        return self._env_build
+
+    def requirements(self):
+        if self.options.with_jpeg:
+            self.requires.add("libjpeg/9d")
+
+    def configure(self):
+        if self.settings.os == "Windows":
+            raise ConanInvalidConfiguration("Zbar can't be built on Windows")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def build(self):
+        env_build = self._configure_autotools()
+        env_build.make()
+
+    def package(self):
+        self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
+        env_build = self._configure_autotools()
+        env_build.install()
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        for dot_la_files in glob.iglob(os.path.join(self.package_folder, "lib", '*.la')):
+            os.remove(dot_la_files)
+
+    def package_info(self):
+        self.cpp_info.libs = tools.collect_libs(self)
+        if self.settings.os == "Linux" and self.options.enable_pthread:
+            self.cpp_info.system_libs = ["pthread"]
+        if tools.is_apple_os(self.settings.os):
+            self.cpp_info.system_libs = ["iconv"]
+            self.cpp_info.system_libs = ["pthread"]
```

Comment:
```suggestion
            if self.options.enable_pthread:
                self.cpp_info.system_libs.append("pthread")
```

Reply:
I did it that way to check if it compiles on mac because I don't have another way of testing it. If it works I'll fix it.